docs/api/profiles.md

```diff
@@ -0,0 +1,86 @@
+# Profiles
+
+Profiles contain information about a User. A User does not need a Profile, but a Profile is always created
+together with a User.
```

Comment:
Yeah a profile is useless without a containing user. So I guess the only way to make a profile is to have it be made together with a user. Unless we need functionality to also retroactively link a new profile to a profile-less user.